packages/manager/apps/dedicated/client/app/account/account.bundle.js

```diff
@@ -12,7 +12,7 @@ import './contacts/service/edit/edit.controller';
 import './contacts/service/edit/edit.routing';
 import './contacts/service/user-contacts-service.controller';
 import './contacts/service/user-contacts-service.routes';
-import './contacts/service/user-contacts.constants';
+import './contacts/user-contacts.constants';
```

Comment:
Why moving this file ? You could also target `./service/user-contacts.constants` at https://github.com/ovh/manager/pull/4065/files#diff-0ceb597b8401081cae880cd32fa48a43f1d6a4b9531e5c0c9fef6b8ca5df26b1R4

Reply:
I thought it's not good practice for a parent module to depend on the child module